OcadoScraper.py

```diff
@@ -8,13 +9,24 @@
 class OcadoScraper:
     def __init__(self):
```

Comment:
I was thinking it might be good to pass categories=ALL into the initialiser, so that we can choose which categories to scrape?

Reply:
Good idea, I was also thinking of some other key work arguments to add to the initialiser (like headless).
We can add a categories=ALL argument when we have functions to get the categories and to access them.

Reply:
Sounds like a plan